Tests/BotTests/MergeService/DispatchServiceContext.swift

```diff
@@ -51,4 +53,9 @@ class DispatchServiceContext {
                 self?.events.append(event)
             }
     }
+
+    static func idleCleanupDelay(times: Double = 1.0) -> DispatchTimeInterval {
```

Comment:
I'm not quite sure why we need to multiply the timeout?

Reply:
It's just to make sure we go _past_ it. We use the same idea already for all other timeouts in our tests, so I figured we should do the same here 🤷‍♂

Reply:
dunno, feels weird to me

Reply:
agreed. updated to remove that multiplication